pandas/core/internals/blocks.py

```diff
@@ -2747,6 +2782,10 @@ def __init__(self, values, placement, ndim=None):
         super(DatetimeBlock, self).__init__(values,
                                             placement=placement, ndim=ndim)
 
+    @property
```

Comment:
this is already in the super class i think

Reply:
Right. This can be removed now that we re-added `DatetimeLikeArray.view`.